Makefile

```diff
@@ -6,7 +6,7 @@ ENV := local
 all: build
 
 # Setup the application
-build: install_deps set_env $(ENV_FILE) collect_static migrate setup_waffle_switches messages
+build: install_deps set_env $(ENV_FILE) assets migrate setup_waffle_switches messages
```

Comment:
Some of the django test workflows use `make collect_static`, you'll need to amend them

Reply:
Whoops, sorted now